eagle-core/eagle-alert-parent/eagle-alert/alert-engine/src/main/java/org/apache/eagle/alert/engine/interpreter/PolicyExecutionPlan.java

```diff
@@ -32,6 +32,11 @@
     private Map<String, List<StreamColumn>> inputStreams;
 
     /**
+     * Actual input streams alias.
+     */
+    private Map<String, String> inputStreamAlias;
```

Comment:
Why need to keep `inputStreamAlias` in `PolicyExecutionPlan`, isn't it only used as local variable in `PolicyExecutionPlannerImpl`?